stock_reserve/__openerp__.py

```diff
@@ -32,6 +32,7 @@
              ],
  'demo': [],
  'data': ['view/stock_reserve.xml',
+          'view/stock_location.xml',
```

Comment:
Here you have one example:

https://github.com/OCA/manufacture/blob/8.0/quality_control_stock/__init__.py#L9

It's launched after installing the module. You can also follows last proposed convention about this: https://github.com/OCA/maintainer-tools/pull/166